.changeset/wise-berries-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+'apollo-angular': minor
```

Comment:
I think this should be a major version, because the behavior changes for subscription testing, and user code must (or should ?) be updated accordingly. What do you think ?

Reply:
I had the same discussion with myself: 
Is it a bug because in production the Subscription would also stay open and this is a misbehavior since Tests should reflect the real behavior?

Is it a feature because it was never specified in any way before? 

I think in both ways it is a breaking-change since an open socket will fail on many tests into timeout errors.